csrc/kernel_cache.cpp

```diff
@@ -1329,6 +1329,9 @@ void FusionKernelRuntime::compileKernel(
   // Running a segment group as a single kernel,
   // make a fusion to run from segmented fusion
   auto fusion_to_run = segmented_fusion_->makeFusion(sg);
+  if (isDebugDumpEnabled(DebugDumpOption::FusionIrPresched)) {
```

Comment:
This prints the concretized and segmented fusion before it is scheduled during compilation.

Reply:
Please update the `FusionIrPresched` option comment in https://github.com/NVIDIA/Fuser/blob/main/csrc/options.h so we know that we're printing the concretized, segmented fusion.